server/src/server.ts

```diff
@@ -1,15 +1,17 @@
 import express, { Application, Request, Response } from "express";
 import cors from "cors";
+import server from "./config/server";
 
+const port = server.port;
+const hostname = server.hostname;
 const app: Application = express();
 app.use(express.json());
 app.use(cors());
-const port = 3000;
 
 app.get("/", (_req: Request, res: Response): void => {
   res.send("Hello there!");
 });
 
-app.listen(port || 5000, (): void => {
-  console.log(`Server running on port ${port}...`);
+app.listen(port || 8001, (): void => {
```

Comment:
The defaulting can be moved to the config file.

Reply:
Did that but forgot to push last changes. It's addressed now.